.pre-commit-config.yaml

```diff
@@ -1,4 +1,17 @@
 repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-byte-order-marker
+    -   id: check-merge-conflict
+    -   id: check-symlinks
+    -   id: check-toml
+    -   id: check-xml
```

Comment:
I don't think this one is necessary. We don't have any XML files in this project.